gymnasium/envs/mujoco/ant_v4.py

```diff
@@ -17,7 +17,7 @@ class AntEnv(MujocoEnv, utils.EzPickle):
             "rgb_array",
             "depth_array",
         ],
-        "render_fps": 20,
```

Comment:
why is this changed

Reply:
I calculate that dynamically in the constructor, since it depends on self.dt and before it was forcing it to always have this value anyway:

Previous code:

assert (
                int(np.round(1.0 / self.dt)) == self.metadata["render_fps"]
            ), f'Expected value: {int(np.round(1.0 / self.dt))}, Actual value: {self.metadata["render_fps"]}'